tvplayer/src/main/java/com/kaltura/tvplayer/OnMediaInterceptorListener.java

```diff
@@ -0,0 +1,5 @@
+package com.kaltura.tvplayer;
+
+public interface OnMediaInterceptorListener {
+    void onComplete();
+}
```

Comment:
Please make this an inner class of `PKMediaEntryInterceptor`, called `Listener`.